packages/backend/README.md

```diff
@@ -55,3 +55,40 @@ The config is passed to `ioredis` - see https://github.com/luin/ioredis#tls-opti
 | REDIS_TLS_CA_FILE_PATH   | Path to CA certificate - overrides well-known CAs. | ""                       |
 | REDIS_TLS_KEY_FILE_PATH  | Path to private key for client authentication.     | ""                       |
 | REDIS_TLS_CERT_FILE_PATH | Path to certificate for client authentication.     | ""                       |
+
+### Design
+
+[Source ->](https://mermaid-js.github.io/mermaid-live-editor/edit#pako:eNp1U8tuwjAQ_JXIBxQk-AEOSI3ooaeCmlObHkyyDRaJTR37ECH-vbt-5AE0B3s9M-udteMrK1UFbMNqzS-nJN8VMsGvs0cPHHl5Bll59E2WqhWyTmOw530L0iw9_W5NrYiOwZw-WGUgdWNAXspSWWnSMM9R2mgMI9d1gHoaA7IH0CkNYZ1nX2kuatAZgGlg-T1J3CHngm6AfQFH-HCgYo_E3fU7aGKjpLlretC4hkngggEmy4TS_FAzWa-3EwNPyOB2bsMxrsyk9KN8hIO3J7uMnY12HeF9z04vWfjjxXnYYzHaXcTkpBG_VlTC9AkPmRNvKJ1cFclxwhukw_xXP_fhWMq-_5Eck2dP1PH-56W38X_xMD4AtmIt6JaLCt_KleCCmROaK9gGw4rrc8EKeUOdvVTcwCv2qTTbGG1hxbg16qOXZVx7zU5wfGVtBC9cfiqFyx_edHD7AzbEK6M)
+<img alt="backend design" src="docs/backend.svg">
+
+### Chart Of Accounts
```

Comment:
I think in this particular instance, because it's development docs, the readme is where I'd like it to live. If we've got things related to running backend or integrating with it, those should live under `/docs/backend`